src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js

```diff
@@ -11,12 +12,16 @@ class GlobalTooltip extends Component {
     window.removeEventListener('mousemove', this.moveTooltip);
   }
 
-  moveTooltip = event => {
-    this.globalTooltip.style.left = `${event.pageX}px`;
-    this.globalTooltip.style.top = `${event.pageY -
-      this.globalTooltip.offsetHeight -
-      15}px`;
-  };
+  moveTooltip = _.throttle(event => {
+    const left = event.pageX;
+    const top = event.pageY;
+    setTimeout(() => {
```

Comment:
yeah, you are right.
Wanna switch this from using top/left to using transforms well your in here? :D